codecs/avif/enc/avif_enc.cpp

```
@@ -35,12 +47,15 @@ struct AvifOptions {
   int tune;
   // 0-50
   int denoiseLevel;
+  // toggles AVIF_CHROMA_DOWNSAMPLING_SHARP_YUV
+  bool enableSharpYUV;
 };
 
 thread_local const val Uint8Array = val::global("Uint8Array");
 
 val encode(std::string buffer, int width, int height, AvifOptions options) {
```

Cppcheck (2.10-2)

[performance] Function parameter 'buffer' should be passed by const reference.

(passedByValue)